var/spack/repos/builtin/packages/augustus/package.py

```diff
@@ -84,6 +84,34 @@ def edit(self, spec, prefix):
             makefile.filter('$(HTSLIB)/libhts.a',
                             '$(HTSLIB)/../lib/libhts.a', string=True)
 
+        # Set compile commands for each compiler and
+        # Fix for using 'boost' on Spack. (only after ver.3.3.1-tag1)
+        if self.version >= Version('3.3.1-tag1'):
+            with working_dir(join_path('auxprogs', 'utrrnaseq', 'Debug')):
+                filter_file('g++',
+                            '{0}'.format(spack_cxx), 'makefile', string=True)
+                filter_file('g++ -I/usr/include/boost', '{0} -I{1}'
+                            .format(spack_cxx,
+                                    self.spec['boost'].prefix.include),
+                            'src/subdir.mk', string=True)
+
+        # Set compile commands to all makefiles.
+        makefiles = [
+            'auxprogs/aln2wig/Makefile',
+            'auxprogs/bam2hints/Makefile',
+            'auxprogs/bam2wig/Makefile',
+            'auxprogs/checkTargetSortedness/Makefile',
+            'auxprogs/compileSpliceCands/Makefile',
+            'auxprogs/homGeneMapping/src/Makefile',
+            'auxprogs/joingenes/Makefile',
+            'src/Makefile'
+        ]
+        for makefile in makefiles:
+            filter_file('gcc', '{0}'
+                        .format(spack_cc), makefile, string=True)
+            filter_file('g++', '{0}'
+                        .format(spack_cxx), makefile, string=True)
```

Comment:
No need to use string formatting here, `spack_cc` and friends are already strings.

Reply:
Thanks for review. `g++` is included regex, so I deleted `string` arg from only `filter_file` func of 110 line.

Reply:
Actually, `string=True` is fine, what I meant was you can do:
```python
filter_file('gcc', spack_cc, makefile, string=True)
```
There's no need to convert `spack_cc` to a string.

Reply:
Sorry... I was misunderstanding. I fixed it.